share/spice/crunchbase/collections/crunchbase_collections.js

```diff
@@ -0,0 +1,66 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_crunchbase_collections = function(api_result){
+        // throw error if no result, error result, no items, or array of 0 items
+        if (!api_result || api_result.error || !DDG.getProperty(api_result, 'data.items') || api_result.data.items.length === 0) {
+            return Spice.failed('crunchbase');
+        }
+        var data = api_result.data, // get org data
+	    items = data.items, // get all orgs
+	    firstResult = items[0], // get first org to render for now
+	    permalink = firstResult.path; // get endpoint of first org
+
+	var metadata = api_result.metadata; // get metadata for base urls
+	// get base urls if metdata exists
+	if (metadata){
+            var wwwPathPrefix = metadata.www_path_prefix,
+                imagePathPrefix = metadata.image_path_prefix;
+	}
+
+        // get organization information from organization endpoint
+	$.getJSON('js/spice/crunchbase/organizations/'+ permalink.split('/')[1],function(response) {
```

Comment:
`item.set()` is now available and can be used here. Amazon.js contains a live example.

My only concern is that (it seems like) we get so little information from the API initially and I'm wondering if we have enough after the first API call to decide whether the result is relevant.

I was initially thinking this would be a tile view that shows multiple IA's but it makes sense to display a single result when we have an exact match.

How does the API behave when you don't query an exact company name? e.g "crunchbase pets"

If it's returning a bunch of irrellevant results, we're probably going to want to add a `relevancy: {}` block to the `Spice.add()` call so we can ensure we don't show anything totally irrellevant.